src/settings/OpenExceptionalForm/ExceptionWrapper.js

```diff
@@ -32,6 +35,10 @@ import {
 
 import { setTimezoneOffset } from '../utils/time';
 
+const EMPTY_STRING = '';
+
+const isValueInvalid = (value) => isNil(value) || value === EMPTY_STRING;
```

Comment:
`_.isEmpty()` works for strings as well as objects and arrays, though regrettably always returns true for numeric arguments. Still, it would simplify this a little since you wouldn't need your own function here and could just check `if (!isEmpty(someField))` in `checkBeforeSave` below. To me, that feels simpler and keeps the check close to where it's being used, which is nice, but this obviously works just fine. Your call.

Reply:
Good point, thanks! Done.